stemcell_builder/stages/bosh_sysctl/apply.sh

```diff
@@ -8,8 +8,14 @@ source "$base_dir/lib/prelude_apply.bash"
 # shellcheck source=../../lib/prelude_bosh.bash
 source "$base_dir/lib/prelude_bosh.bash"
 
-cp "$dir/assets/60-bosh-sysctl.conf" "$chroot/etc/sysctl.d"
-chmod 0644 "$chroot/etc/sysctl.d/60-bosh-sysctl.conf"
+# shellcheck disable=SC2154
+if [ "${stemcell_operating_system}" == "ubuntu" ] || \
+   [ "${stemcell_operating_system_version}" == "7" ] || \
+   [ "${stemcell_operating_system}" == "rhel" ]; then
```

Comment:
Since we don't care about centos and this gets installed on both ubuntu and rhel, this condition should be removed.